From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Fri, 6 Nov 2009 09:19:58 +0100
Subject: [PATCH] reactivate check for tty

The busybox crew switched of the test for a tty in silentoldconfig,
we use this feature to break when oldconfig needs interaction
in automated build.

This patch reactivates this feature.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 scripts/kconfig/conf.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/kconfig/conf.c b/scripts/kconfig/conf.c
index 9befa2b..e65e25c 100644
--- a/scripts/kconfig/conf.c
+++ b/scripts/kconfig/conf.c
@@ -503,7 +503,7 @@ int main(int ac, char **av)
 			break;
 		case 's':
 			input_mode = ask_silent;
-			valid_stdin = isatty(0); //bbox: && isatty(1) && isatty(2);
+			valid_stdin = isatty(0) && isatty(1) && isatty(2);
 			break;
 		case 'd':
 			input_mode = set_default;
-- 
1.7.1

